coral-hive/build.gradle

```diff
@@ -46,7 +48,7 @@ generateGrammarSource {
   ]}
 
 artifacts {
-  archives jar, javadocJar, sourcesJar
+  archives jar
 }
```

Comment:
I think you shouldn't need to add the `jar` here either, since you're using `from components.java` in your `publications` block. The `java` plugin should be adding that for you automatically (same comment for all the other modules)